sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -315,7 +317,7 @@ Text Analytics for health is a containerized service that extracts and labels re
 unstructured texts such as doctor's notes, discharge summaries, clinical documents, and electronic health records.
 Currently, Azure Active Directory (AAD) is not supported in the Healthcare recognition feature. In order to use this 
 functionality, request to access public preview is required. For more information see [How to: Use Text Analytics for health][healthcare].
```

Comment:
Looks like the snippet about AAD not being supported for healthcare and the part about requesting access for use should be removed